### DISCUSSION TITLE
Upgrade not working v0.17.4 Stable to AzuraCast 0.19.0

### DISCUSSION THREAD

**Discussion by ebinvthomas** [AzuraCast/AzuraCast#7235] [Q&A]
We are unable to upgrade Azuracast v0.17.4  to AzuraCast 0.19.0; some stations are not able to migrate; I mean, stop playing.

**Comment by BusterNeece** [comment]
This should probably be an issue reported instead of a discussion, but the other question is why are you updating to 0.19.0 when that's not the latest version?

Be sure, if you do create an issue, to include logs on the stations that won't start. They may have custom Liquidsoap code that's causing trouble and needs updating.
